index.js

```diff
@@ -1,4 +1,13 @@
-export const CircleMode = require('./lib/modes/CircleMode');
-export const DragCircleMode = require('./lib/modes/DragCircleMode');
-export const DirectMode = require('./lib/modes/DirectModeOverride');
-export const SimpleSelectMode = require('./lib/modes/SimpleSelectModeOverride');
+"use strict";
+
+Object.defineProperty(exports, "__esModule", { value: true });
+
+const CircleMode = require("./lib/modes/CircleMode");
+const DragCircleMode = require("./lib/modes/DragCircleMode");
+const DirectMode = require("./lib/modes/DirectModeOverride");
+const SimpleSelectMode = require("./lib/modes/SimpleSelectModeOverride");
+
+exports.CircleMode = CircleMode;
+exports.DirectMode = DirectMode;
+exports.DragCircleMode = DragCircleMode;
+exports.SimpleSelectMode = SimpleSelectMode;
```

Comment:
This isn't supposed to be:

```suggestion
module.exports = {
  CircleMode,
  DirectMode,
  DragCircleMode,
  SimpleSelectMode
}
```